### PR TITLE
To add --all install option to setup.

### PR DESCRIPTION
**Pull request by FyodorK**
For now setup.py has two option --tf and --torch
Both intended to install tensorflow and pytorch parts of the product separately
But EXTRAS_REQUIRE also has 'all' label that suppose to install both tensorflow and torch at the same time.
The option '--all' covers using of this label.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -106,6 +106,10 @@ def find_version(*file_paths):
     INSTALL_REQUIRES.extend(EXTRAS_REQUIRE["tf"])
     sys.argv.remove("--tf")
 
+if "--all" in sys.argv:
```
Comment (alexsu52) [inline setup.py:109]: @FyodorK @vshampor Do you really need it? I mean we can use tf and torch options at the same time.

Comment (FyodorK) [reply setup.py:109]: 1. this option is needed for bdist installation
2. In case p.1 is questionable -> why this option has implemented in EXTRAS_REQUIRE dict but still not used at all?

**Review by l-bat — APPROVED** [review]

**Review by alexsu52 — APPROVED** [review]
